geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraQueryPlan.scala

```diff
@@ -75,5 +76,5 @@ case class StatementPlan(
   ) extends CassandraQueryPlan {
 
   override def scan(ds: CassandraDataStore): CloseableIterator[Row] =
-    CassandraBatchScan(ds.session, ranges, numThreads)
+    CassandraBatchScan(this, ds.session, ranges, numThreads, ds.config.queries.timeout.map(Timeout.apply))
```

Comment:
Is there any chance that the C* unit tests would let us exercise timeouts?

Reply:
i think we'd have the same issue in any of the data stores, in that it's requires timing and any unit tests that rely on timing tend to fail occasionally.

Reply:
Grrr...  "I wonder how it'd be to 'fix' the unit test framework....", he said trailing off in a manner to indicate that this was a bad idea....